lib/developer_portal/app/views/developer_portal/main_layout.html.liquid

```diff
@@ -22,6 +22,9 @@
     {{ '/javascripts/vendor/colorbox/jquery.colorbox-patched.js' | javascript_include_tag }}
     {{ '/javascripts/3scale_v2.js' | javascript_include_tag }}
     {{ '/javascripts/excanvas.compiled.js' | javascript_include_tag }}
+     {% if provider.payment_gateway_type == :stripe %}
+      <script src="https://js.stripe.com/v3" async></script>      
```

Comment:
We should provide this dependency, not relying on their CDN.

Reply:
In fact it is recommended doing it this way:

> Include the Stripe.js script on each page of your site—it should always be loaded directly from https://js.stripe.com, rather than included in a bundle or hosted yourself.

Reply:
Yes, that's the recommended way of doing it, according to stripe. But I remember we had to move all the cdn assets of the Dev Portal to our codebase, because some clients sometimes have restrictions on their networks (on-premises).

Reply:
Yes, we support disconnected install so bundling script would be better. (although if somebody would want to use billing they would need internet connection anyways)